### PR TITLE
guide: nav follow-ups

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Follow-up to https://github.com/iterative/dvc.org/pull/4011

- ~~Update existing links~~
- [x] More links around PCP
- [x] Consistent URLs, names, and frontmatter for new how-to pages

### CODE REVIEW

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Repurposed to do misc. link/ naming updates instead of the existing links -- that one is going to be huge so I want it to have a single focus... ⏳

File: src/components/DownloadButton/index.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { logEvent } from '@dvcorg/gatsby-theme-iterative/src/utils/front/plausib
 import * as styles from './styles.module.css'
 import { OS, useUserOS } from '../../utils/front/useUserOS'
 
-const VERSION = `2.30.0`
+const VERSION = `2.29.0`
```
Comment (shcheklein) [inline src/components/DownloadButton/index.tsx:12]: weird change

Comment (jorgeorpinel) [inline src/components/DownloadButton/index.tsx:12]: Not sure how this happened during some merge from `main`. Reverting...
```suggestion
const VERSION = `2.30.0`
```

**Comment by jorgeorpinel** [comment]
OK solved conflicts and sanity checked the changes once more. Merging!

**Pull request merged**